prompt-service/src/unstract/prompt_service/variable_extractor/prompt_variable_service.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import re
+from typing import Any
+
+from utils.request import HTTPMethod, make_http_request
+
+from .constants import VariableConstants, VariableType
+
+logger = logging.getLogger(__name__)
+
+
+class VariableService:
+
+    @staticmethod
+    def replace_static_variable(
+        prompt: str, structured_output: dict[str, Any], variable: str
+    ) -> str:
+        output_value = VariableService.check_static_variable_run_status(
+            structure_output=structured_output, variable=variable
+        )
+        static_variable_marker_string = "".join(["{{", variable, "}}"])
+        if not output_value:
+            return prompt
```

Comment:
```suggestion
        if not output_value:
            return prompt
        static_variable_marker_string = "".join(["{{", variable, "}}"])
        
```
NIT: Exiting from the function early

Reply:
Returning the `prompt` here, if the variable referred is not executed.

Reply:
@harini-venkataraman I reordered the statements so that we exit early and don't wastefully perform this
`static_variable_marker_string = "".join(["{{", variable, "}}"])`